src/app/components/projects/components/site-card/site-card.component.scss

```diff
@@ -17,11 +17,6 @@ li.list-group-item {
   }
 }
 
-img {
-  max-width: 60px;
-  height: auto;
-}
-
```

Comment:
why remove css definition?

Reply:
This was no longer needed as we tell the image directive that the image should be 60px, and it automatically sets the height to auto